src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -2494,6 +2494,211 @@ .maxstack  1
         }
 
         [Fact]
+        public void TestMiscOperators()
+        {
+            var text = @"
+using System;
+
+public class C
+{
+    public void M(ref S global)
+    {
+        S local1 = stackalloc int[10];
+        global = local1++; // error 1
+
+        Span<int> local2 = stackalloc int[10];
+        global = local2; // error 2
+
+        Console.Write(global == local2);
+
+        global += local1;
+        local1 += global;
+    }
+    public static void Main()
+    {
+    }
+}
+public ref struct S
+{
+    public static implicit operator S(Span<int> s) => throw null;
+    public static S operator ++(S s) => throw null;
+    public static bool operator ==(S s1, Span<int> s2) => throw null;
+    public static bool operator !=(S s1, Span<int> s2) => throw null;
+    public override bool Equals(object o) => throw null;
+    public override int GetHashCode() => throw null;
```

Comment:
Minor point: Consider removing `Equals` and `GetHashCode` from these tests if not needed.